### ISSUE TITLE
[TW-336] task reference [concept]

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#371]
_David Patrick on 2012-08-17T03:35:38Z says:_ 

 Spawned from Penelope Fudds desire for a more concise reference, as discussed around feature #1068, *task ref* is a mini help system. Organised by concept; write-commands, attributes, modifiers, dates, color, etc, *task ref [concept]* produces a very compact reference to the subject, ideally in a single line of text. For example

 % tw ref reports
active,all,blocked,completed,newest,next,oldest,overdue,projects,ready,recurring,tags,unblocked,waiting

might be all that's needed to jog your memory, or 

 % tw ref write-commands
add,annotate,append,delete,denotate,done,duplicate,edit,modify,prepend,start,stop,undo

could be all the reference you need, at least till you memorise all of taskwarriors many features.

 % tw ref list

lists the available reference.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2012-08-17T03:35:38Z
Modified: 2014-02-09T00:50:20Z
```

**Comment by taskwarrior** [comment]
_David Patrick on 2012-08-17T04:05:03Z says:_ 

 The output of task ref might be similar to that of various helper commands, but presented horizontally. It might make sense to actually call some helper commands using the reference command, like *task ref _projects*, to get the same reference in a single line, without all the vertical space.

**Comment by taskwarrior** [comment]
_David Patrick on 2013-05-04T23:41:48Z says:_ 

 this is a functional duplicate of #1068
